index.d.ts

```diff
@@ -0,0 +1,52 @@
+declare module '@articulate/authentic' {
+  interface VerifyOpts {
+    algorithms?: string[]
+    audience?: string | RegExp | Array<string | RegExp>
+    clockTimestamp?: number
+    clockTolerance?: number
+    issuer?: string | string[]
+    ignoreExpiration?: boolean
+    ignoreNotBefore?: boolean
+    jwtid?: string
+    subject?: string
+    maxAge?: string
+  }
+
+  interface JWKSOpts {
+    cache?: boolean,
+    rateLimit?: boolean
+  }
+
+  interface AuthenticOpts {
+    issWhitelist: string[],
+    jwks?: JWKSOpts,
+    verify?: VerifyOpts
+  }
+
+  interface Validator {
+    (token: string): Promise<Authentic.JWT>
+  }
+
+  function Authentic(opts: AuthenticOpts): Validator
+
+  namespace Authentic {
+    interface JWT {
+      // Standard JWT claims
+      aud: string,
+      cid?: number,
+      exp: number,
+      iat: number,
+      iss: string
+      jti?: number,
+      scp?: Array<String>
+      sub: string
+      uid?: number,
```

Comment:
TIL both commas and semicolons are valid delimiters in interface declarations

Reply:
I actually prefer the no comma look. Thanks for pointing out the inconsistency :)